src/components/footer/footer.tsx

```diff
@@ -2,10 +2,18 @@ import './footer.css'
 
```

Comment:
```suggestion
footerImages = {
         "instagram":{
               "link":"hiperlink",
               "label":"Instagram"
         }
}
```
Algo asi para hacer mas legible el codigo. El label puede NO estar, pero los links si son muy grandes no dejan ver las otras propiedades del elemento html